docs/src/main/sphinx/admin/properties-catalog.md

```diff
@@ -0,0 +1,43 @@
+# Catalog management properties
+
+(prop-catalog-management)=
+
+## `catalog.management`
+
+- **Type:** {ref}`prop-type-string`
+- **Allowed values:** `static`, `dynamic`
+- **Default value:** `static`
+
+Trino reads catalog property files and configures available catalogs on server
+startup. When set to `dynamic`, catalog configuration can also be managed using
+[](/sql/create-catalog) and [](/sql/drop-catalog). Creating and dropping
+catalogs using these SQL commands will create new and delete existing catalog
+property files on all nodes.
+
+:::{warning}
+This feature is experimental only. Because of the security implications the
+syntax might change and be backward incompatible.
+:::
+
+:::{warning}
```

Comment:
Do we really still need this. I thought this was a problem from the Hadoop dependencies and those are all gone. And even if .. its only applicable for LOTS of create/drop cycles .. that should be rare.

Reply:
I was testing this a few days ago and there was a 10% increase in non-heap memory usage after creating and dropping a hive catalog 15 times.

Reply:
Lets leave it for now .. but we have to keep an eye on it and remove it when it no longer applies